docs/rules/vue-recommended/element-attribute-order.md

```diff
@@ -6,3 +6,50 @@ Checks if the component always uses `is`, `v-for`, `v-if`, `v-else-if`, `v-else`
 ## ❓ Why it's good to follow this rule?
 
 - **Consistency:** It makes the code consistent and easier to maintain.
+
+## 😱 Examples of code for which this rule will throw a warning
+
+::: warning
+The attributes of elements (including components) should be ordered consistently. The following examples illustrate cases where the attribute order is incorrect.
+:::
+
+### Example 1: Incorrect Attribute Order
+```javascript
+<template>
+    <input v-on:input="handleInput" v-model="inputValue">
+</template>
+```
+
+In this example, the attributes of the `<input>` element are out of order. The `v-on:input` directive should come after `v-model` to maintain consistent attribute ordering.
+
+### Example 2: Incorrect Attribute Order in a Different Element
+
+```javascript
+<template>
+  <div id="app" v-if="isVisible" ref="myDiv" v-on:click="handleClick"></div>
```

Comment:
v-on should be @

Reply:
fixed in https://github.com/rrd108/vue-mess-detector/pull/133/commits/c7841567f5b1592e0473650c19a6b5dc5ccc4a71